docs/api/java/getting-started.md

```diff
@@ -1,78 +1,70 @@
 # Getting started
 
-:::warning
-
-- We are currently updating the Java SDK and you may encounter errors when trying things out.
-- Apologies for the inconvenience, we will update the documentation in due course.
-  :::
-
 In this guide we will demonstrate how to:
 
 1. Connect to the zkSync network.
 2. Deposit assets from Ethereum into zkSync.
-3. Transfer and withdraw funds (native and ERC20 tokens).
-4. Deploy a smart contract.
-5. Interact with any smart contract.
+3. Check balances.
+4. Transfer and withdraw funds (native and ERC20 tokens).
+5. Deploy a smart contract.
+6. Deploy a smart contract with `create2`.
 
-## Prerequisite
+## Prerequisites
 
 This guide assumes that you are familiar with the basics of [Java](https://docs.oracle.com/en/java/) programming language.
 
-## Installation
+## Dependency
 
-To install the zkSync Java SDK, just add the following dependency:
+For connecting ZkSync2 library just add the following dependency to your build file.
 
 Maven `pom.xml`
 
-```gradle
+```xml
 <project>
   ...
   <dependencies>
     <dependency>
       <groupId>io.zksync</groupId>
       <artifactId>zksync2</artifactId>
-      <version>0.0.1</version>
+      <version>0.1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.web3j</groupId>
+      <artifactId>core</artifactId>
+      <version>4.9.7</version>
     </dependency>
   </dependencies>
 </project>
-
 ```
 
 Gradle `build.gradle`
 
-```
+```groovy
 dependencies {
-    implementation "io.zksync:zksync2:0.0.1"
+    implementation "io.zksync:zksync2:0.1.1"
 }
 ```
```

Comment:
if these are two different options to install it, we can use :::code-tabs

Reply:
Yes they are two different options. I have tried that but the problem is that they are not marked so if I try to use @tab dependencies its not working